hi-ml-cpath/src/SSL/data/cxr_datasets.py

```diff
@@ -191,7 +191,7 @@ def _prepare_dataset(self) -> None:
         mapping = {0: 0, 3: 0, 1: 1, 2: 1}
         # For monitoring purpose with use binary classification CV03vsCV12
         self.dataset_dataframe["final_label"] = self.dataset_dataframe.final_label.apply(lambda x: mapping[x])
-        self.indices = np.arange(len(self.dataset_dataframe))
+        self.indices = np.arange(len(self.dataset_dataframe)).tolist()
```

Comment:
Why are these changes needed? Is it because of the new NumPy version?

Reply:
`mypy` finds different errors with the new Python versions. We declared indices as being a list, but `mypy` so far did not complain about assigning an `ndarray` to that list. Now it does.